bust/exceptions.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, bus_type):
 
 
 class InvalidResetMode(RuntimeError):
-    """Documentation for InvalidResetMode
+    """Documentation for InvalidResetMode.
```

Comment:
maybe also add actual documentation 
```suggestion
    """ @brief Raised when an invalid reset mode is set.
```